blueprints/addon/additional-package.json

```diff
@@ -11,7 +11,7 @@
     "ember-try": "^2.0.0"
   },
   "peerDependencies": {
-    "ember-source": "^3.28.0 || ^4.0.0"
+    "ember-source": "^4.4.0"
```

Comment:
I think it would make sense for this to remain "^4.0.0" even if only testing newer 4.x. It would be a shame for updating addons to break in projects only because of the peerDep requirements.

Reply:
I understand the concern, but that would also mean we need to update the Compatibility section in the README to be in sync with this I think?

AFAIK, we've always used the oldest LTS version in the README:

```
* Ember.js v4.4 or above
* Ember CLI v4.4 or above
```

Reply:
IIRC having a peerDependencies section is relatively new. The README has matched what the ember-try scenarios have had. I think it will cause unnecessary pain when addons update the blueprint but otherwise do not make changes and still remain compatible with older versions, even if the peerDeps say otherwise. It's really on the addon author to not make that bump without a major bump but I'm not sure how many will realize that. Once again, I wish we could put comments in package.json. 

cc @ef4 for an opinion.